uint32 exposure_usecs
uint32 exposure_time_ratio
